content/kubermatic/master/advanced/addons/_index.en.md

```diff
@@ -55,12 +54,70 @@ To deploy the changes:
 helm upgrade --install --wait --timeout 300 --values values.yaml --namespace kubermatic kubermatic charts/kubermatic
 ```
 
-##### Setting a custom docker registry
+##### Setting a Custom Docker Registry
 
 In case you want to set a custom registry for all addons, you can specify the `-overwrideRegistry` flag on the `kubermatic-controller-manager` or via the helm setting `kubermatic.controller.overwriteRegistry`.
 It will set the specified registry on all control plane components & addons.
 
-### How to add a custom addon
+### Accessible Addons
+Accessible addons can be installed in each user-cluster in Kubermatic on user demand. If addon is both default
+and accessible then it will be installed in user-cluster but also visible to the user, who can manage it from the UI
+like other accessible addons. The accessible addons are:
+
+* [node-exporter](https://github.com/prometheus/node_exporter): Exports metrics from the node
+
+Accessible addons can be managed in the UI from the cluster details view:
+
+![View](/img/advanced/addons/view.png)
+
+#### Configuration
+To configure which addons shall be accessible, set the following settings in the `values.yaml` for the kubermatic chart:
+
+```yaml
+kubermatic:
+  api:
+      # List of optional addons that can be installed into every user-cluster. All need to exist in the addons image.
+      accessibleAddons:
+      - node-exporter
+```
+
+Accessible addons are configured by the `AddonConfig` custom resources with the same names as the addons that are
+configured. The configuration is not required, but it is recommended. Each config should contain logo and description.
+Form specification of addon variables is optional. Here is an example of `node-exporter` config:
+
+```yaml
+apiVersion: kubermatic.k8s.io/v1
+kind: AddonConfig
+metadata:
+  name: node-exporter
```

Comment:
Maybe include a namespace if the AddonConfig is not cluster-wide? (Is it cluster-wide or namespaced?)

Reply:
It is cluster-wide.